_posts/2021-05-16-overview-of-architecture-review-processes.md

```diff
@@ -0,0 +1,253 @@
+  1. The Architecture Review Board (ARB) which is a standing team that oversees the review process and its effect on the organization.
+
+Moreover, there are further roles in the overall process - depending on the size of the enterprise and if these roles are required:
+  1. There is a review client which is often the project team. The review client pays for the development or is the architecture review’s sponsor.
+  1. The project members who present the architecture to the review team in the process.
+  1. The project management encompasses all the managers responsible for the project’s success.
+  1. The review angel is selected by the ARB being responsible to work with the project team addressing any organizational or political issues that may arise.
+  1. The ARB chair is the architecture review process advocate. The ARB chair is responsible for ensuring the effectiveness of the review.
+
+### Process
+
+The overall review process bases on a few general principles such as that a clearly defined problem statement as the basis of the architecture, reviews are open processes, and companies conduct reviews for the project’s benefit (see also: {% cite Maranzano2005 %} for the full list of five principles).
+
+The review process follows roughly four phases.
+The specific implementation may vary.
+  1. Screening Phase: The project team requests a review. The project staff and ARB review the request to determine whether a review would benefit the project. If a review should be conducted, the ARB selects a review angel (see also: previous section) to oversee the project’s review process.
+  1. Preparation Phase: The ARB selects a review team and a review leader. Togehter with the project team, they determine the number of reviews and the initial review’s date and agenda. The review team, project team, review leader, and review angel verify that the project that should be reviewed has a clearly defined problem statement driving the architecture and an appropriate documentation as a basis for the review.
+  1. Review Meeting Phase: During the review, the members of the project present the mentioned problem statement and how the proposed architecture solves it. The reviewers ask further questions in order to figure out issues they believe could make the project an incomplete or inadequate solution to the problem. Finally, the review team meets privately and generates a report. The report is presented to the project team. Furthermore, management alerts can be raised, etc.
+  1. Follow-Up Phase: The review team delivers the report with the review’s findings to the project team within certain time frame such as 15 days after the review meetings are done. The project team (and, if a management alert has been raised, the management) must respond within two weeks.
+
+For more details, we refer to {% cite Maranzano2005 %}.
+
+### Artifacts
+
+As artifacts, Maranzano et al. mention three different types of artifacts {% cite Maranzano2005 %}:
```

Comment:
```suggestion
Three different types of artifacts are getting created:
```

Sure, it was Maranzano et al. who defined it. However, it is your article ;)

Another thought: I am reading this article mainly because of the topic. Not because of the Author (Maranzano).